hack/tools/fetch-images/main.go

```diff
@@ -307,6 +307,8 @@ func getImagesForChart(info *ChartInfo) ([]string, error) {
 	images.ExtraImagesFiles = info.extraImagesFiles
 	// kubeVersion needs to be set for some addons
 	images.KubeVersion = "v1.29.0"
+	// apiVersions needs to be set for some addons
+	images.APIVersions = []string{"snapshot.storage.k8s.io/v1"}
```

Comment:
Support for APIVersions was added in `github.com/d2iq-labs/helm-list-images v0.12.0`